inference-engine/src/mkldnn_plugin/nodes/ctc_greedy_decoder.cpp

```diff
@@ -0,0 +1,148 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "base.hpp"
+#include "ie_parallel.hpp"
+
+#include <vector>
+#include <string>
+
+namespace InferenceEngine {
+namespace Extensions {
+namespace Cpu {
+
+class CTCGreedyDecoderImpl: public ExtLayerBase {
+public:
+    explicit CTCGreedyDecoderImpl(const CNNLayer* layer) : mergeRepeated_(true) {
+        std::string errPrefix = "CTCGreedyDecoder layer with name '" + layer->name + "' ";
+        if (layer->insData.size() != 2)
+            THROW_IE_EXCEPTION << errPrefix << "has invalid number of input edges: " << layer->insData.size();
+        if (layer->outData.size() != 1)
+            THROW_IE_EXCEPTION << errPrefix << "has invalid number of outputs edges: " << layer->outData.size();
+
+        auto inData = layer->insData[DATA_INDEX].lock();
+        auto sequenceLenData = layer->insData[SEQUENCE_LENGTH_INDEX].lock();
+        if (!inData || !sequenceLenData)
+            THROW_IE_EXCEPTION << errPrefix << "has nullable inputs.";
+        if (inData->getTensorDesc().getPrecision() != Precision::FP32)
+            THROW_IE_EXCEPTION << errPrefix << "has unsupported 'data' input precision: " << inData->getTensorDesc().getPrecision();
+        if (sequenceLenData->getTensorDesc().getPrecision() != Precision::FP32)
+            THROW_IE_EXCEPTION << errPrefix << "has unsupported 'sequence_length' input precision: " << sequenceLenData->getTensorDesc().getPrecision();
+
+        std::vector<DataConfigurator> inputConfigs{{ConfLayout::PLN, Precision::FP32}, {ConfLayout::PLN, Precision::FP32}};
+        std::vector<DataConfigurator> outputConfigs{{ConfLayout::PLN, Precision::FP32}};
+        addConfig(layer, inputConfigs, outputConfigs);
+
+        mergeRepeated_ = layer->GetParamAsBool("ctc_merge_repeated", true);
```

Comment:
As I can see this parameter name from legacy https://github.com/openvinotoolkit/openvino/blob/master/docs/MO_DG/prepare_model/convert_model/Legacy_IR_Layers_Catalog_Spec.md#CTCGreadyDecoder
After migration on opset1 it will be changed on ```merge_repeated```
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/CTCGreedyDecoder_1.md

Could you add changes like this: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/non_max_suppression.cpp#L33

Reply:
fixed